var/spack/repos/builtin/packages/py-pytorch-lightning/package.py

```diff
@@ -16,6 +16,7 @@ class PyPytorchLightning(PythonPackage):
 
     license("Apache-2.0")
 
+    version("2.0.7", sha256="632dec9af8036f726904e691c505d7650658ef0f4054a062c9e6a940ca09dfd8")
```

Comment:
I think you forgot to replace the sha256 after copying the directive for `2.0.0`.

I cannot confirm this sha256.  I'm getting `4e6bc1e1f7b0c69016ea2fe0616b18fa62bd3d8661c7ff02c11d317746cfc5f5`, which matches the one posted on pypi.org.

Reply:
Good catch, thanks!